lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -202,6 +227,7 @@ ReportRenderer.GroupJSON; // eslint-disable-line no-unused-expressions
  *     timing: {total: number},
  *     initialUrl: string,
  *     url: string,
+ *     lighthouseRunWarnings: !Array<string>,
```

Comment:
not sure how we should be modeling extensions to the report JSON here, the latest renderer needs to render older versions that might be missing runWarnings, which you account for, but it *is* nice to have the closure annotations match the current invariants

thoughts?

Reply:
yeah, it should be optional. Fixed